python/tank/platform/engine.py

```diff
@@ -1504,14 +1568,15 @@ def _create_dialog_with_widget(self, title, bundle, widget_class, *args, **kwarg
             
         Additional parameters specified will be passed through to the widget_class constructor.
         """
+
         # get the parent for the dialog:
         parent = self._get_dialog_parent()
         
         # create the widget:
         widget = self._create_widget(widget_class, *args, **kwargs)
-        
+
         # apply style sheet
-        self._apply_external_styleshet(bundle, widget)        
```

Comment:
I did a quick grep and it is being called from tk-maya. So sadly it looks like we have maintain a backwards compatibility pathway due to a typo that I think I must have introduced. Sorry guys! But would be great if we could hide the misspelt version in sphinx at least :)